website/content/docs/lambda/registration.mdx

```diff
@@ -0,0 +1,279 @@
+interface, or Cloud Development Kit (CDK):
+
+1. Add the `aws_lambda_function` resource to your Terraform configuration and specify the name of the Lambda function.
+1. Add a `tags` block to the resource and specify the tags you want to use to register the function (refer to [Supported Tags](#supported-tags).
+
+In the following example, the `example` Lambda function is registered using the `enabled`, `payload-passthrough`, and `invocation-mode` tags:
+
+```hcl
+resource "aws_lambda_function" "example" {
+  …
+  function_name = "lambda"
+  tags = {
+    "serverless.consul.hashicorp.com/v1alpha1/lambda/enabled" = "true"
+    "serverless.consul.hashicorp.com/alpha/lambda/payload-passthrough" = "true"
+    "serverless.consul.hashicorp.com/alpha/lambda/invocation-mode" = "ASYNCHRONOUS"
+  }
+}
+```
+
+#### Supported Tags
+
+The following tags are supported. In all cases, the `<PREFIX>` should be
+`serverless.consul.hashicorp.com/v1alpha1/lambda`. For example,
+`serverless.consul.hashicorp.com/v1alpha1/lambda/enabled`.
+
+| Tag | Description |
+| - | - |
+| `<PREFIX>/enabled` | Determines if Lambda registrator will sync the Lambda into Consul. |
+| `<PREFIX>/payload-passthrough` | Determines if the body Envoy receives is converted to JSON or directly passed to Lambda. This attribute is optional and defaults to `false`. |
+| `<PREFIX>/invocation-mode` | Specifies the [Lambda invocation mode](https://docs.aws.amazon.com/lambda/latest/operatorguide/invocation-modes.html) Consul uses to invoke the Lambda. This tag will default to `SYNCHRONOUS`, but also supports `ASYNCHRONOUS` invocations. |
+| `<PREFIX>/namespace` | <EnterpriseAlert inline />This tag specifies the Consul namespace the service will be registered in. If `enterprise` is enabled on Lambda registrator, this defaults to `default`. |
+| `<PREFIX>/partition` | <EnterpriseAlert inline />This tag specifies the Consul partition the service will be registered in. If `enterprise` is enabled on Lambda registrator, this defaults to `default`. |
+| `<PREFIX>/aliases` | This tag specifies a `+`-separated string of Lambda aliases that will be registered into Consul. If this is set to `dev+staging+prod`, the `dev`, `staging`, and `prod` aliases of the Lambda function will be registered into Consul. |
+
+## Manual Configuration
+You can manually register Lambda functions if you are unable to automate the process using the Lambda registrator.
+### Register the Service
+
+`<SERVICE_NAME>` specifies the Consul service name for the Lambda function.
+Replace `<SERVICE_NAME>` and store the following in `lambda.json`:
+```json
+{
+  "Node": "lambdas",
+  "SkipNodeUpdate": true,
+  "NodeMeta": {
+    "external-node": "true",
+    "external-probe": "true"
+  },
+  "Service": {
+    "Service": "<SERVICE_NAME>"
+  }
+}
+```
+
```

Comment:
```suggestion
1. Create a configuration for registering the service. You can copy the following example and replace `<SERVICE_NAME>` with your Consul service name for the Lambda function:
    <CodeBlockConfig filename="lambda.json">
    ```json
    {
          "Node": "lambdas",
          "SkipNodeUpdate": true,
          "NodeMeta": {
            "external-node": "true",
            "external-probe": "true"
          },
          "Service": {
            "Service": "<SERVICE_NAME>"
          }
        }
    ```
1. Save the configuration to `lambda.json`.
```
Does this work better as sequential steps?